Microsoft.Identity.Web.UI/Areas/MicrosoftIdentity/Controllers/AccountController.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Web.UI.Areas.MicrosoftIdentity.Controllers
+{
+    [NonController]
+    [AllowAnonymous]
+    [Area("MicrosoftIdentity")]
+    [Route("[area]/[controller]/[action]")]
+    public class AccountController : Controller
+    {
+        private readonly IOptionsMonitor<MicrosoftIdentityOptions> _options;
+
+        public AccountController(IOptionsMonitor<MicrosoftIdentityOptions> azureADOptions)
+        {
+            _options = azureADOptions;
+        }
+
+        [HttpGet("{scheme?}")]
+        public IActionResult SignIn([FromRoute] string scheme)
```

Comment:
The action results can receive a scheme name, allowing the multiple schemes scenario